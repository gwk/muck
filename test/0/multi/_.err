muck note: c-many.txt: `python3 c-many.txt.py` runningâ€¦
muck note: a-to-many-0.txt: `python3 'a-to-many-{i}.txt.py' 0` runningâ€¦
muck note: a-to-many-0.txt: `python3 'a-to-many-{i}.txt.py' 0` finished: (via open).
muck note: a-to-many-0.txt: product [1mis new[22m; 12 B.
muck note: a-to-many-1.txt: product [1mis new[22m; 12 B.
muck note: b-through-0.txt: `python3 'b-through-{i}.txt.py' 0` runningâ€¦
muck note: b-through-0.txt: `python3 'b-through-{i}.txt.py' 0` finished: (via stdout).
muck note: b-through-0.txt: product [1mis new[22m; 12 B.
muck note: b-through-1.txt: `python3 'b-through-{i}.txt.py' 1` runningâ€¦
muck note: b-through-1.txt: `python3 'b-through-{i}.txt.py' 1` finished: (via stdout).
muck note: b-through-1.txt: product [1mis new[22m; 12 B.
muck note: c-many.txt: `python3 c-many.txt.py` finished: (via stdout).
muck note: c-many.txt: product [1mis new[22m; 0 B.
