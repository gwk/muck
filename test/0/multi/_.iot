{
  'cmd': '$MUCK -no-times -deps c-many.txt',
  'code': 0,
  'links': {
    'test/0/multi/a-to-many-{i}.txt.py',
    'test/0/multi/b-through-{i}.txt.py',
    'test/0/multi/c-many.txt.py',
  },
  'files': {
    '_build/c-many.txt' : {
      'val': '',
    },
  },
}
