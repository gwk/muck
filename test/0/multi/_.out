
a-to-many-0.txt[m (â‡  [31mb-through-0.txt[m, [31mc-many.txt[m)
â”—â•¸[32ma-to-many-{i}.txt.py[m

a-to-many-1.txt[m (â‡  [31mb-through-1.txt[m, [31mc-many.txt[m)
â”—â•¸[32ma-to-many-{i}.txt.py[m

b-through-{i}.txt.py[m (â‡  [32mb-through-0.txt[m, [32mb-through-1.txt[m)

c-many.txt[m
â”¡â•¸[32mc-many.txt.py[m
â”œâ•´[31ma-to-many-0.txt[m â‡¡
â”œâ•´[31ma-to-many-1.txt[m â‡¡
â”œâ•´[31mb-through-0.txt[m
â”‚ â”¡â•¸[32mb-through-{i}.txt.py[m â‡¡
â”‚ â””â•´[31ma-to-many-0.txt[m â‡¡
â””â•´[31mb-through-1.txt[m
  â”¡â•¸[32mb-through-{i}.txt.py[m â‡¡
  â””â•´[31ma-to-many-1.txt[m â‡¡
